package.json

```diff
@@ -64,6 +64,7 @@
     "react-moveable": "^0.21.6",
     "react-photo-gallery": "^8.0.0",
     "react-pure-render": "^1.0.2",
+    "react-transition-group": "^4.4.1",
```

Comment:
Confused, why is this green here when it's already a dependency?

On `main`:

https://github.com/google/web-stories-wp/blob/e816b2eec6aa4951580d6848e88206deb0414105/package.json#L67

Reply:
Oh I didn't realize this was already a dep on main. I'll update.

It's adding it as an added dep cus I took it out on the branch I have this one diffing against, cus I added and removed it not knowing it was already there